ui/server/src/main/scala/pl/touk/nussknacker/ui/process/processingtypedata/ProcessingTypeDataReader.scala

```diff
@@ -29,6 +29,15 @@ object ProcessingTypeDataReader extends LazyLogging {
         key -> config.as[ProcessingTypeConfig](key)(ProcessingTypeConfig.reader)
       }.toMap
     }
-    config.as[Map[String, ProcessingTypeConfig]]("processTypes")
+
+    val processTypesOption = config.getAs[Map[String, ProcessingTypeConfig]]("processTypes")
+    val scenarioTypesOption = config.getAs[Map[String, ProcessingTypeConfig]]("scenarioTypes")
+    (scenarioTypesOption, processTypesOption) match {
+      case (Some(scenarioTypes), _) => scenarioTypes
+      case (None, Some(processTypes)) =>
+        logger.info("ScenarioTypes configuration is missing - falling back to old configuration")
```

Comment:
maybe warn with clear message: it will be removed in next version?

Reply:
Done